### ISSUE TITLE
[no-loss-of-precision] False negative with numeric separators

### ISSUE THREAD

**Issue by cherryblossom000** [typescript-eslint/typescript-eslint#2082]
**Repro**
```json
{
  "parser": "@typescript-eslint/parser",
  "rules": {
    "no-loss-of-precision": 2
  }
}
```

```ts
1_0
```

**Expected Result**
No error.

**Actual Result**
```
/path/to/file.ts
  1:1  error  This number literal will lose precision at runtime  no-loss-of-precision

✖ 1 problem (1 error, 0 warnings)
```

**Additional Info**
Should there be a `@typescript-eslint/no-loss-of-precision` rule?

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `3.0.0` |
| `@typescript-eslint/parser`        | `3.0.0` |
| `TypeScript`                       | `3.7.5` |
| `ESLint`                           | `7.1.0` |
| `node`                             | `12.16.2` |
| `npm`                              | `6.14.4` |

**Comment by bradzacher** [comment]
Duplicate of #2076
